recipes/ncvis/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "ncvis" %}
+{% set version = "1.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c015d84b954d60d5a0985c8709b8dad7ef6274ae9b3cd1f41786ca2ba6f05224
+
+build:
+  number: 0
+  skip: true  # [py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux and not aarch64]
+  host:
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux and not aarch64]
+    - python
+    - pip
+    - numpy
+    - cython
+  run:
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux and not aarch64]
+    - python
+    - numpy
```

Comment:
Please see https://conda-forge.org/docs/maintainer/knowledge_base.html?#building-against-numpy for the proper way to link agains numpy

Reply:
Fixed it, thank you